README.md

```diff
@@ -89,6 +89,7 @@ With this exercise we are attempting to collect the GitHub accounts of all stude
 
   #### Spring 2018 Class
   * Joe Jeppi: [github.com/jjeppi](https://github.com/jjeppi)
+  * Walter Alvarez: [github.com/WalterAlvarez92](https://github.com/WalterAlvarez92)
```

Comment:
@WalterAlvarez92 ... bring the line starting with `https` up so that everything is on *one* line

Reply:
I brought the line up, but dont know if i have the most u to date

Reply:
*up

Reply:
Where is the commit for bringing the line up?

Reply:
position change